### PR TITLE
Address review comments to #1249

### PR DESCRIPTION
**Pull request by ahoppen**

### CODE REVIEW

**Comment by ahoppen** [comment]
@swift-ci Please test

File: Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift [hunk 1]
```diff
@@ -406,7 +415,7 @@ extension SwiftPMBuildSystem: SKCore.BuildSystem {
       return [ConfiguredTarget(targetID: "", runDestinationID: "dummy")]
     }
 
-    if url.pathExtension == "h", let target = try? target(forHeader: path) {
+    if let target = try? target(forHeader: path) {
```
Comment (bnbarham) [inline Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift:418]: Ah, I noticed the comment in the other PR before this one:
> Did you also rename `target(forHeader:)`? Maybe `inferTarget(for:)`?

**Review by bnbarham — APPROVED** [review]

**Review by hamishknight — APPROVED** [review]

File: Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift (hunk 1 above)
Comment (ahoppen) [reply Sources/SKSwiftPMWorkspace/SwiftPMBuildSystem.swift]: Renamed to `inferredTarget(for:)`.

**Comment by ahoppen** [comment]
@swift-ci Please test

**Comment by ahoppen** [comment]
@swift-ci Please test Windows

**Pull request merged**